src/simtools/applications/simulate_prod.py

```diff
@@ -122,6 +122,17 @@ def _parse(description=None):
         required=False,
         default=False,
     )
+    config.parser.add_argument(
+        "--sequential",
+        help=(
+            "If set to true, run CORSIKA and sim_telarray instances in sequential order "
```

Comment:
Is this actually correct? One cannot set this parameter to 'set to true', but only set the flag 'sequential'. Suggestion simply "Run CORSIKA and sim_telarray instances in ...".

Further down, I suggest to remove the sentence "This mode is useful particularly on the grid, "
            "where typically we request a single core per job.".  or replace it by a more technical description (without `we` - who is that?). e.g. `Enables single-core mode; otherwise, CORSIKA and sim_telarray run in parallel. `

Reply:
You are right, I wrote this when the argument was different and also not in a technical enough way. I took your suggested sentence almost as is.

Reply:
All good - fine with me.